README.md

```diff
@@ -158,7 +158,8 @@ This is a complete example of a task definition:
 {
     "label": "Run Hollywood Main script",
     "type": "shell",
-    "command": "${config:hollywood.exePath} ${workspaceFolder}/${config:hollywood.mainFile}",
+    "command": "${config:hollywood.exePath}",
+    "args":["${workspaceFolder}/${config:hollywood.mainFile}"],
```

Comment:
As you mentioned it in #1 could you update your PR and delete the ${workspaceFolder}/ string? Thank you. I think this is easier to merge then (I am not super experienced with GitHub PRs but I think this is the correct workflow. Correct me, if I am wrong).